addons/@mlabs-haskell/godot-cardano/src/abstract.gd

```diff
@@ -0,0 +1,8 @@
+class_name Abstract
```

Comment:
I think this could prove useful, the only limitation I see is that it extends `Node`, so it wouldn't work for any abstract class that extends `RefCounted` or `Object` (of which we have a few I think).

Unfortunately, I don't think there is a way of having abstract classes with different bases without some code repetition (`AbstractNode`, `AbstractRefCounted`, etc). Maybe it's not too bad?

Reply:
Let me know tomorrow about these holes, I am curious.